lib/cmock_generator_plugin_callback.rb

```diff
@@ -79,10 +79,4 @@ def mock_interfaces(function)
     lines << "  Mock.#{func_name}_CallbackBool = (int)0;\n"
     lines << "  Mock.#{func_name}_CallbackFunctionPointer = Callback;\n}\n\n"
   end
-
-  def mock_verify(function)
-    func_name = function[:name]
-    "  if (Mock.#{func_name}_CallbackFunctionPointer != NULL)\n    call_instance = CMOCK_GUTS_NONE;\n"
```

Comment:
Why do you believe that this line isn't necessary?

Reply:
These two lines appear to be intended to defeat the "called fewer times" check (the same check that I optimized in the other commit) when a callback is installed, in the same way that Function_Ignore defeats that check.

The problem is that the callback plugin has later priority and runs after the expect plugin.  So the "called fewer times" check has actually already run at this point.  So the value assigned to call_instance here isn't used, not by that check nor by anything else that I can see.

Anyway, I don't think we even want the callback plugin to defeat the "called fewer times" check.  At least, if you call Function_AddCallback() and Function_Expect(), but then don't call Function(), that should be an error.  Not sure about what should happen when you call Function_Stub() and then Function_Expect() -- that combination doesn't make a lot of sense to me.